nbs/course/lesson7-superres.ipynb

```diff
@@ -1206,6 +1206,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "dblock = DataBlock(blocks=(ImageBlock, ImageBlock),\n",
+    "                   get_items=get_image_files,\n",
+    "                   get_y=lambda x: path_hr/x.name,\n",
+    "                   splitter=RandomSplitter(),\n",
+    "                   item_tfms=Resize(size),\n",
+    "                   batch_tfms=[*aug_transforms(max_zoom=2.), Normalize.from_stats(*imagenet_stats)])\n",
     "dbunch_mr = dblock.dataloaders(path_mr, bs=1, val_bs=1, path=path,\n",
     "                             batch_tfms=[*aug_transforms(), Normalize.from_stats(*imagenet_stats)])         \n",
```

Comment:
You need to remove the `batch_tfms` from there, they should only be in the dblock.

Reply:
Thank you for pointing it out. I raised a new PR: https://github.com/fastai/fastai2/pull/273